app/portainer/components/stack-duplication-form/stack-duplication-form-controller.js

```diff
@@ -72,5 +77,24 @@ angular.module('portainer.app').controller('StackDuplicationFormController', [
         ctrl.formValues.endpoint.Id === ctrl.currentEndpointId
       );
     }
+
+    function initView() {
+      var endpointMode = ctrl.applicationState.endpoint.mode;
+  
+      StackService.stacks(
+        true,
+        endpointMode.provider === 'DOCKER_SWARM_MODE' && endpointMode.role === 'MANAGER',
```

Comment:
Stack name validation should be done against all endpoints for all type of providers, you can simply avoid the provider/role check here:

```suggestion
        true,
```

Reply:
I have changed it to true. however, it seems to me that there is an issue when docker isn't in swarm mode. We try to load the services and we get a 503.